exercises/pangram/pangram_test.rb

```diff
@@ -19,6 +19,16 @@ def test_pangram_with_only_lower_case
     assert Pangram.is_pangram?(str)
   end
 
+  def test_pangram_with_underscore_character
+    str = 'the quick brown_fox jumps over the lazy dog'
+    assert Pangram.is_pangram?(str)
+  end
+
+  def test_pangram_with_sentence_containing_numbers
+    str = 'the qu1ck br0wn fox jumps over the lazy dog'
```

Comment:
Is that the purpose of the test?  

I don't think it is, and if so, it is not expressed clearly.

It appears that the purpose is that if there are 26 unique characters and yet it is missing the only letter this is missing, which happens to be an 'i', it should not be classified as a pangram.  

For intent, though, we would definitely need to ask @victorbrender since he is the only one that can confirm intent, as the author.

The visual distraction of similarly shaped numbers in the place of letters is simply a distraction.  Noting that the 'i' is the only thing missing is the exercise of detail attention that you speak of.  It happens here regardless of the visual distraction of the numbers similarity to the letters they are replacing.